synapse/replication/tcp/resource.py

```diff
@@ -205,6 +205,13 @@ async def _run_notifier_loop(self) -> None:
                             # send, so we send a `POSITION` to inform other
                             # workers of the updated position.
 
+                            # We skip this for the `caches` stream as a) it
+                            # generates a lot of traffic as every worker would
+                            # echo each write, and b) nothing cares if a given
+                            # worker's caches stream position lags.
+                            if stream.NAME == CachesStream.NAME:
```

Comment:
Should we log that we're skipping it?

Reply:
I think this is going to get called *a lot*.